### PR TITLE
Feature logging #6

### PR DESCRIPTION
**Pull request by yamikuronue**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
 const fs = require('fs-promise');
+const dateFormat = require('dateformat');
+
 
 module.exports = {
-    handle: function() {
-        return fs.appendFile('/home/rover/hooksreceived.log', 'got file');
+    handle: function(body) {
+        const zen = body.zen;
+        const timestamp = dateFormat(new Date(), 'mmm dd yyyy HH:MM:ss');
```
Comment (RaceProUK) [inline index.js:9]: I'd prefer dates in ISO format, unless there's a specific requirement for a different format

Comment (yamikuronue) [reply index.js:9]: Really? I find it harder to parse in a hurry or when flustered if there's no spaces between the portions.

Comment (RaceProUK) [reply index.js:9]: I was thinking the variant with hyphens i.e. yyyy-MM-dd HH:MM:ss

Comment (yamikuronue) [reply index.js:9]: Let's take this to slack and I'll adjust the format in my next PR if that's okay? :)

**Review by RaceProUK — APPROVED** [review]

**Pull request merged**